wp-content/themes/debtcollective/src/scss/components/molecules/impactful-callout.scss

```diff
@@ -0,0 +1,36 @@
+div.is-style-callout {
```

Comment:
@robu9  - I'm merging this and it's great.
Pea added some class names that were a little shorter, let's favor those rather than-the-very-long-wordpress-ones